bpf/include/vmlinux.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
-#ifndef __VMLINUX_H__
-#define __VMLINUX_H__
+#ifndef __VMLINUX_GENERIC_H__
```

Comment:
Since 2 generated headers come both with `__VMLINUX_H__` include guard, rename this one that is easier to modify.